README.rst

```diff
@@ -247,10 +247,38 @@ The LSP plugin has to be enabled in the Kate (version >= 19.12.0) plugins menu:
 This assumes that nimlsp was installed through nimble.
 *Note: Server initialization may fail without full path specified, from home directory, under the ``"command"`` entry, even if nimlsp is in system's ``PATH``.*
 
+VS Code
+-------
+Install a VS Code extension that supports NimLSP (2 available at the moment):
+
+- https://marketplace.visualstudio.com/items?itemName=junknet.nimlsp
+- https://marketplace.visualstudio.com/items?itemName=bung87.vscode-nim-lsp
+
+Set ``nimlsp.path`` extension setting to the binary path of ``nimlsp``. If you've installed ``nimlsp`` using nimble it is already in system's ``PATH``.
+
+``stderr`` of ``nimlsp`` process will be available in ``Output > nim|nimlsp`` terminal
+
+
 Run Tests
 =========
 Not too many at the moment unfortunately, but they can be run with:
 
 .. code:: bash
 
     nimble test
+
+
+Debug
+=========
+Use ``nimlsp_debug`` executable instead of ``nimlsp``, which is installed alongside it and should already be available in your path. 
+
+Log files containing stdin/out will be generated in ``getTempDir() / "nimlsp-" & $getCurrentProcessId() / "nimlsp.log"`` folder, where ``getCurrentProcessId()`` is the running pid of ``nimlsp_debug`` instance executed by the IDE/extension, and can be read using ``pgrep -a nimlsp_debug``. Crashes may print stacktraces in stderr, which is not contained in logs but may captured by LSP client.
+
+when stdin/out/err is not enough, it is possible to trace all system calls of ``nimlsp[_debug]`` via ``sudo strace -p<pid> -s9999 > strace.log 2>&1``
+
+``test/logrunner`` can be used to replay the query submitted to ``nimlsp`` stored inside nimlsp.log:
+
+.. code:: bash
```

Comment:
Did you intend for this section to just be the help output of the logrunner application? If so then maybe `bash` isn't the right code highlighter.